src/components/AboutPageSideNavBar/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState } from 'react';
+import { CommunityNavigationSection } from '../../types';
+import NavigationItem from '../NavigationItem';
+
+const commNavSections: CommunityNavigationSection[] = [
+  {
+    title: 'About',
+    sections: [],
+  },
+  {
+    title: 'Project Governance',
+    sections: [
```

Comment:
These sections are not needed. See the original screenshot.

Reply:
Okay.

Reply:
Where can we find the original screenshot?
Is this the same as nodejs,org.about menu?

Reply:
Yes it is. https://github.com/nodejs/nodejs.dev/issues/1354#issue-899083518

Reply:
Done.